android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -80,10 +84,15 @@ private Boolean isTablet() {
     constants.put("buildVersion", "not available");
     constants.put("buildNumber", 0);
 
+    WifiManager manager = (WifiManager) this.reactContext.getSystemService(Context.WIFI_SERVICE);
+    WifiInfo wifiInfo = manager.getConnectionInfo();
+    String ipAddress = Formatter.formatIpAddress(wifiInfo.getIpAddress());
+    String macAddress = wifiInfo.getMacAddress();
```

Comment:
Just remembered that these will change over time so will have to create a function that will be called for a callback.